js/actions/bookmarkActions.js

```diff
@@ -4,15 +4,18 @@
 
 'use strict'
 
+const Immutable = require('immutable')
 const siteUtil = require('../state/siteUtil')
 const appActions = require('./appActions')
 const windowActions = require('./windowActions')
 const eventUtil = require('../lib/eventUtil')
+const appStoreRenderer = require('../stores/appStoreRenderer')
 const {SWITCH_TO_NEW_TABS} = require('../constants/settings')
 const getSetting = require('../settings').getSetting
 
 const bookmarkActions = {
-  openBookmarksInFolder: function (allBookmarkItems, folderDetail) {
+  openBookmarksInFolder: function (folderDetail) {
```

Comment:
in any case if `tabs.create` doesn't encapsulate all the logic for creating a tab then we need to fix that because the api is not useful if it can only be called through `APP_CREATE_TAB_REQUESTED`